pyquil/paulis.py

```diff
@@ -170,22 +170,22 @@ def copy(self):
         return new_term
 
     @property
-    def program(self):
+    def program(self) -> Program:
         return Program([QUANTUM_GATES[gate](q) for q, gate in self])
 
-    def get_qubits(self):
+    def get_qubits(self) -> List[int]:
         """Gets all the qubits that this PauliTerm operates on.
         """
         return list(self._ops.keys())
 
-    def __getitem__(self, i):
+    def __getitem__(self, i: int) -> str:
         return self._ops.get(i, "I")
 
-    def __iter__(self):
+    def __iter__(self) -> Iterable[Tuple[int, str]]:
```

Comment:
```suggestion
    def __iter__(self) -> Iterator[Tuple[int, str]]:
```

mypy complains about `Iterable` not having a `__next__` attr. This actually [returns an `Iterator`](https://docs.python.org/3/library/stdtypes.html#generator-types).

Reply:
Fixed.